bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/internal/servlets/ChildrenEditorServlet.java

```diff
@@ -0,0 +1,108 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.sandbox.internal.servlets;
+
+import java.io.IOException;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(
+    service = Servlet.class,
+    property = {
+        "sling.servlet.methods=POST",
+        "sling.servlet.resourceTypes=sling/servlet/default",
```

Comment:
should this really be bound to all resourcetypes? As a side effect of the intended usecase it can be used to remove all (known) childnodes of a resource (of course within the boundaries of the ACLs), or reorder childnodes etc.

Reply:
For now we could bind he servlet to the Carousel resouce type. For the long term, once the new Container component is available, we could make the Carousel extend the Container and bind the servlet to the Container resource type.

Reply:
@jckautzmann Would binding the servlet to Carousel RT work with the proxy-resource pattern?

Reply:
@vladbailescu You're right, this needs to be tested first.